corehq/apps/smsforms/app.py

```diff
@@ -56,9 +61,20 @@ def start_session(domain, contact, app, module, form, case_id=None, yield_respon
         }
     
     if app and form:
```

Comment:
Before we merge this we should make sure that the tests in the following files pass (they must be run manually by uncommenting them from the corresponding tests/__init__.py file):

corehq.apps.sms.tests.inbound_handlers
corehq.apps.kookoo.tests.outbound

Instructions for how to setup the test environment are here: https://github.com/dimagi/commcare-hq/blob/6b864322d159cb0f0d498eea57d234eeea6abd15/corehq/apps/sms/tests/util.py#L43